sirepo/template/srw.py

```diff
@@ -319,7 +318,10 @@ def extract_report_data(filename, model_data):
     if _DATA_FILE_FOR_MODEL[rep_name]['dimension'] == 3:
         width_pixels = int(model_data['models'][orig_rep_name]['intensityPlotsWidth'])
         scale = model_data['models'][orig_rep_name]['intensityPlotsScale']
-        info = _remap_3d(info, allrange, file_info[filename][0][3], file_info[filename][1][2], width_pixels, scale)
+        rotate_angle = model_data['models'][orig_rep_name]['rotateAngle']
+        rotate_reshape = model_data['models'][orig_rep_name]['rotateReshape']
```

Comment:
For both rotate_angle and rotate_shape, defaulting the values to 0 and '0' would allow a report to be created without requiring these values. One example is the mirrorReport which appears on the beamline mirror editor and doesn't have many report fields.

```python
        rotate_angle = model_data['models'][orig_rep_name].get('rotateAngle', 0)
        rotate_reshape = model_data['models'][orig_rep_name].get('rotateReshape', '0')
```

Reply:
Thanks for your updates!